tests/ui/test_login.py

```diff
@@ -1,23 +1,24 @@
+import os
 import pytest
 
 from pages.desktop.home import Home
 
 
-@pytest.mark.skip(
-    reason='https://github.com/mozilla/addons-server/issues/2462')
-def test_login(base_url, selenium, user):
+@pytest.mark.skipif(os.environ.get('PYTEST_BASE_URL') is None,
+                    reason='Live Server login currently not functioning')
```

Comment:
> my_base_url starts the live server for each test.

right but no test is actually using it right now, right? Just want to make sure I'm not missing anything.